src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,12 +1,44 @@
+import { useEffect } from "react";
 import { WishlistItems } from "../../components";
+import { useWishlist, useAuth } from "../../context";
+import { fetchInitialWishlistService } from "../../services";
 import "./Wishlist.css";
 
 export const Wishlist = () => {
+  const { wishlistState, wishlistDispatch } = useWishlist();
+  const {
+    authState: { token },
+  } = useAuth();
+
+  const getInitialWishlist = async () => {
+    try {
+      const response = await fetchInitialWishlistService(token);
+      if (response.status === 200) {
+        wishlistDispatch({
+          type: "GET_INITIAL_WISHLIST",
+          payload: response.data.wishlist,
+        });
+      } else {
+        throw new Error();
+      }
+    } catch (err) {}
```

Comment:
You're catching the error but doing nothing with it. You could maybe `console.log()` it or send an `alert()` to the user

Reply:
Done